x/spec/types/proposal.go

```diff
@@ -7,7 +7,7 @@ import (
 	sdkerrors "cosmossdk.io/errors"
 )
 
-func checkSpecProposal(spec Spec) error {
+func (spec Spec) ValidateBasic() error {
```

Comment:
add check that contributors addresses are valid

Reply:
this is done in the ValidateSpec() method